kumascript/macros/WebExtAPISidebar.ejs

```diff
@@ -57,15 +57,15 @@ function buildSublist(pages, title, ignoreBadges) {
     var pageBadges = '';
     if (!ignoreBadges) {
 
-        if (hasTag(aPage, 'Experimental')) {
+        if (aPage.status.includes("experimental") || hasTag(aPage, 'Experimental')) {
```

Comment:
Note that this is all disabled by default - i.e. the function this is called in `buildSublist` is only ever called with `ignoreBadges` set to true. To test this I had to modify a couple of the cases where the method is called (and then modify the docs themselves).